.github/workflows/python-api-k2.yaml

```diff
@@ -1,10 +1,10 @@
 # TODO: Merge this with allenNLP to have a single workflow for all docker images.
```

Comment:
Would you mind adding a second workflow as this one? https://github.com/huggingface/api-inference-community/blob/main/.github/workflows/python-api-flair-cd.yml 

That will enabled continuous deployment